### PR TITLE
Setup codequality

### PR DESCRIPTION
**Pull request by dory1318**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ the below**
 
 [You can find the engineering project outline here.](https://github.com/makersacademy/course/tree/master/engineering_projects/rails)
 
-2. The card wall is here: <please update>
+2. The card wall is here: https://trello.com/b/qGXi1GtK/acebook
```
Comment (mateowh) [inline README.md:10]: This is linked to in it's own section at the bottom of the readme. To avoid repetition, can we remove this one?

Comment (dory1318) [reply README.md:10]: We talked with Lima that the link should be here and the one in the bottom should be deleted

Comment (mateowh) [reply README.md:10]: Great - thanks, I've updated that.

File: README.md [hunk 2]
```diff
@@ -24,14 +24,23 @@ First, clone this repository. Then:
...
 > bin/rails server # Start the server at localhost:3000
 ```
+### To run code quality checks, type the following commands from console:
+more infos: https://github.com/whitesmith/rubycritic
+```
+rubycritic --no-browser
+```
+more infos: https://github.com/flyerhzm/rails_best_practices
+```
+rails_best_practices .
```
Comment (mateowh) [inline README.md:34]: Running this produces one warning (with this explanation https://rails-bestpractices.com/posts/2011/08/19/restrict-auto-generated-routes/). Perhaps we could address this before putting this into master - would you like me to take a look, or would you like us to pair and fix it?

Comment (dory1318) [reply README.md:34]: Fixing it in pairs would be great!

**Review by mateowh — APPROVED** [review]

**Pull request merged**